packages/components/src/internal/components/domainproperties/DomainRowExpandedOptions.tsx

```diff
@@ -65,7 +66,12 @@ export class DomainRowExpandedOptions extends React.Component<Props> {
             queryName,
         } = this.props;
 
-        switch (field.dataType.name) {
+        let dataTypeName = field.dataType.name;
```

Comment:
Might be worth a comment what is going on here with the preprocessing before the switch.

Reply:
done